addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -758,6 +758,14 @@ registry.Parallax = Animation.extend({
         this.visibleArea.push(this.visibleArea[0] + this.$el.innerHeight() + this.viewport);
         this.ratio = this.speed * (this.viewport / 10);
 
+        const parallaxType = this.$el[0].dataset.parallaxType;
+        this.isZoomIn = parallaxType === "zoom_in";
+        this.isZoomOut = parallaxType === "zoom_out";
+        if (this.isZoomIn || this.isZoomOut) {
+            this.zoomLevel = parseFloat(this.$el[0].dataset.scrollBackgroundRatio || 1.2); 
```

Comment:
There is `this.el` you can use instead of `this.$el[0]`

Reply:
cc @SergeBayet :wink:

Reply:
I change it everywhere, right?

Reply:
Only in code you are adding (our owl-related team will get rid of the rest soon, no need for extra diff here)